torch/_inductor/config.py

```diff
@@ -165,6 +165,28 @@
 # Enable indirect_indexing asserts for decompositions and lowerings
 debug_index_asserts = False
 
+# Provides a way to toggle the enforcement of NHWC fallbacks for convolutions
+# this is necessary if they are less performant
+if torch.version.hip:
+
+    if torch.cuda.is_available():
+        gpu_name = torch.cuda.get_device_name(0)
+    else:
+        gpu_name = ""
+
+    if "MI2" in gpu_name:
```

Comment:
@jataylo I think [PR 107477](https://github.com/pytorch/pytorch/pull/107477) should provide a more robust way of coding such GPU-type-conditional checks since I'm not sure if the `device_properties.name` field will always contain a reliable string to match on.

Reply:
Planning to replace with gcnArchName when that PR lands

Reply:
Actually, to delink the PRs, let's go ahead with above logic slightly modified to match "MI2\d\d"